.github/workflows/ci_linux_x64_nogil-libshortfin.yml

```diff
@@ -0,0 +1,97 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: CI - libshortfin - GIL disabled
```

Comment:
Can we call this something like `Python 3.13 Freethreaded`

Reply:
Of course :)